src/pages/iou/WaypointEditor.js

```diff
@@ -119,6 +133,25 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
                 onBackButtonPress={() => {
                     Navigation.goBack(ROUTES.getMoneyRequestDistanceTabRoute(iouType));
                 }}
+                shouldShowThreeDotsButton={shouldShowThreeDotsButton}
+                threeDotsAnchorPosition={styles.threeDotsPopoverOffset(windowWidth)}
```

Comment:
@Nikhil-Vats could you kindly teach me why we need this prop? Why does `HeaderWithBackButton` need to know the width of the screen?

Reply:
okay that makes sense thank you for your explanation 🙇